src/messages.py

```diff
@@ -0,0 +1,262 @@
+                    "value": "python_help",
+                },
+                {
+                    "name": "ruby",
+                    "text": "Ruby",
+                    "type": "button",
+                    "value": "ruby_help",
+                },
+
+            ]
+        },
+        {
+            "text": "",
+            "fallback": "",
+            "color": "#3AA3E3",
+            "callback_id": "greeting_buttons",
+            "attachment_type": "default",
+            "actions": [
+                {
+                    "name": "suggestion",
+                    "text": "Are we missing something? Click!",
+                    "type": "button",
+                    "value": "suggestion",
+                },
+            ]
+        }
+    ]
+}
+
+SUGGESTION_MODAL = {
+    "callback_id": "suggestion_modal",
+    "title": "Help topic suggestion",
+    "submit_label": "Submit",
+    "elements": [
+        {
+            "type": "text",
+            "label": "Suggestion",
+            "name": "suggestion",
+            "placeholder": "Underwater Basket Weaving"
+        },
+    ]
+}
+
+MENTOR_REQUEST_MODAL = {
+    "callback_id": "mentor_request",
+    "title": "Mentor Service Request",
+    "submit_label": "Submit",
+    "elements": [
+        {
+            "value": "test",
+            "type": "text",
+            "label": "Slack User Name",
+            "name": "Slack User",
+            "placeholder": ""
+        },
+        {
+            "value": "test@test.com",
+            "type": "text",
+            "subtype": "email",
+            "label": "Email",
+            "name": "Email",
+            "placeholder": ""
+        },
+        {
+
+            "type": "select",
+            "label": "Service Type",
+            "name": "service",
+            "placeholder": "Choose a service type",
+            "options": [
+                {
+                    "label": "General Guidance - Voice Chat",
+                    "value": "General Guidance - Voice Chat"
+                },
+                {
+                    "label": "General Guidance - Slack Chat",
+                    "value": "General Guidance - Slack Chat"
+                },
+                {
+                    "label": "Pair Programming",
+                    "value": "Pair Programming"
+                },
+                {
+                    "label": "Code Review",
+                    "value": "Code Review"
+                },
+                {
+                    "label": "Mock Interview",
+                    "value": "Mock Interview"
+                },
+                {
+                    "label": "Resume Review",
+                    "value": "Resume Review"
+                },
+
+            ]
+        },
+        {
+            "type": "select",
+            "label": "Mentor Skillset",
+            "name": "skillset",
+            "optional": "true",
+            "placeholder": "Choose a service type",
+            "options": [
```

Comment:
Lots of these things could be generalized by building the Json with a list of languages. And having a single create modal function call where we can insert the dictionary values.